test/end-to-end/offer.js

```diff
@@ -983,6 +976,24 @@ describe("Offer endpoint tests", () => {
                     expect(extracted_data).toContainEqual(prepared_test_offer);
                 });
             });
+
+            describe("Search for offer requirements", () => {
```

Comment:
This is not search, but normal fetching. You can rename to simply "Offer Requirements"